.github/workflows/test-library.yml

```diff
@@ -22,6 +22,30 @@ on:   # yamllint disable-line rule:truthy
     - 'version-check.py'
     - 'MANIFEST.in'
     - '.github/workflows/test-library.yml'
+  workflow_dispatch:
+    inputs:
+      release-version:
```

Comment:
@abhinavsingh once you merge this PR and set up secrets and envs, you'll be able to go to https://github.com/abhinavsingh/proxy.py/actions/workflows/test-library.yml, click a manual trigger button, type in the desired version, and kick-off a workflow that would eventually get the tested dists all the way to PyPI. Well, almost. It'll pause before running the job that does actual publishing and wait for your approval, then after you'll give it a go-ahead, it'll finish what it started. Before clicking the approval, you can try out/preview the pre-published release from TestPyPI (if you want to do this, of course) that will happen unconditionally, in parallel.